Makefile

```diff
@@ -59,7 +59,8 @@ test: prometheus manifests generate fmt vet envtest ## Run tests with Prometheus
 	@echo "Starting Prometheus for testing..."
 	@$(PROMETHEUS) --config.file=pkg/testconfig/prometheus.yml & \
 	trap 'killall prometheus; rm -rf data' EXIT; \
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... -coverprofile cover.out
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" ginkgo run -p 1 ./... --coverprofile cover.out -test.v --procs 1 --vv --output-interceptor-mode=none
```

Comment:
Ack.

Reply:
@neerajb do you mean the parallelism of 1 or the entire test run using ginkgo CLI? 

This is from the ginkgo docs. I found it easy to debug when running the tests with ginkgo CLI in terms of output separation etc. 

```Under the hood, ginkgo is simply calling go test. While you can run go test instead of the ginkgo CLI, Ginkgo has several capabilities that can only be accessed via ginkgo. We generally recommend users embrace the ginkgo CLI and treat it as a first-class member of their testing toolchain.```

Reply:
I meant using 'go test' instead of using ginkgo cli. While currently all tests written using ginkgo, it might not be true in future.

Reply:
Let's continue with ginkgo CLI until then?

Reply:
Sure. I don't see too much downside to it, if its helping with better test output.